Makefile

```diff
@@ -25,6 +25,7 @@ test-unit:
 	@echo "Reports will be written to ${ARTIFACT_DIR}"
 	python -m pytest tests/unit --cov=app --cov=src --cov=utils --cov-report term-missing --cov-report json:${ARTIFACT_DIR}/coverage_unit.json --junit-xml=${ARTIFACT_DIR}/junit_unit.xml
 	python scripts/transform_coverage_report.py ${ARTIFACT_DIR}/coverage_unit.json ${ARTIFACT_DIR}/coverage_unit.out
+	hack/codecov.sh
```

Comment:
should we do this for the integration job too?

Reply:
also the file needs to be made executable based on the job failure error

Reply:
So we need to get this PR: https://github.com/openshift/release/pull/46818 in as pre-requisite.
Since we do not have any integration tests at this moment I am not uploading its report. But open to other opinions too.

Reply:
we do have integration tests: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_lightspeed-service/50/pull-ci-openshift-lightspeed-service-main-integration/1735377879656566784

Reply:
Updated the same for integration tests too.